l10n_ro_account_report/wizard/d394.py

```diff
@@ -76,29 +76,15 @@ def _get_datas(self, cr, uid, ids, context=None):
         tax_obj = self.pool.get('account.tax')
         xmldict = {}
         user = self.pool.get('res.users').browse(cr, uid, uid)
-        employee = False
-        employee = self.pool.get('hr.employee').search(
-            cr, uid, [('user_id', '=', uid)])
         function = email = ''
-        if employee:
-            employee = self.pool.get(
-                'hr.employee').browse(cr, uid, employee[0])
-            if employee.job_id:
-                function = employee.job_id.name
-            # if employee.partner_id.email:
-            #    email = employee.partner_id.email
-            uid_name = employee.name.split()
-            if len(uid_name) == 2:
-                uid_fname = uid_name[0]
-                uid_name = uid_name[1]
-            else:
-                uid_fname = uid_name[0]
-                for i in [0, len(uid_name) - 1]:
-                    uid_name1 = uid_name[i]
-                uid_name = uid_name1
-
-        if function == '':
-            function = 'Administrator'
+        if user.partner_id.function:
+            function = user.partner_id.function
+        else:
+            raise osv.except_osv(_('Error!'), _('You need to define your Job Position.'))
+        uid_name = user.partner_id.name.split()
+        uid_fname = uid_name[0]
+        uid_name = ' '.join(uid_name[1:])
```

Comment:
nume = nume de familie
prenume = nume mic. 
nu se pune adrian vasile in loc de vasile adrian?